fluent-dom/src/localization.js

```diff
@@ -55,17 +37,22 @@ export default class Localization {
    */
   async formatWithFallback(keys, method) {
     const translations = [];
+
     for (let ctx of this.ctxs) {
       // This can operate on synchronous and asynchronous
       // contexts coming from the iterator.
       if (typeof ctx.then === "function") {
         ctx = await ctx;
       }
-      const errors = keysFromContext(method, ctx, keys, translations);
-      if (!errors) {
-        break;
+      const missingIds = keysFromContext(method, ctx, keys, translations);
+
+      if (typeof console !== "undefined") {
+        const locale = ctx.locales[0];
+        const ids = Array.from(missingIds).join(", ");
```

Comment:
With `missingIds` being a nullable set, this should first check if `missingIds !== null`. I'd prefer if `keysFromContext` always returned a set, even an empty one.